src/NetscriptFunctions/Sleeve.ts

```diff
@@ -17,7 +17,7 @@ import { SleeveWorkType } from "../PersonObjects/Sleeve/Work/Work";
 import { canAccessBitNodeFeature } from "../BitNode/BitNodeUtils";
 
 export const checkSleeveAPIAccess = function (ctx: NetscriptContext) {
-  if (Player.bitNodeN !== 10 && !Player.sourceFileLvl(10)) {
+  if (Player.bitNodeN !== 10 && Player.activeSourceFileLvl(10) <= 0) {
```

Comment:
This one is weird. How do sleeves work if you have 10 disabled? From the other place, they don't get *destroyed*, but are they disabled?

If you can still use them manually, you should be able to use them via API.